packages/backend/src/query/collectColumnReferences.test.ts

```diff
@@ -1,28 +1,30 @@
-import { QueryResult, col } from '@synthql/queries';
+import { QueryResult, col, query } from '@synthql/queries';
 import { describe, expect, test } from 'vitest';
-import { DB, from } from '../tests/generated.schema';
+import { DB } from '../tests/generated';
 import { city, film } from '../tests/queries.v2';
 import { AnyQuery } from '../types';
 import { collectColumnReferences } from './collectColumnReferences';
 import { printPath } from '../util/path/printPath';
 import { getIn } from '../util/tree/getIn';
 
+const from = query<DB>().from;
+
 describe('collectColumnReferences', () => {
     describe('collectColumnReferences', () => {
         const cases: Array<{
             query: AnyQuery;
             expected: Array<string>;
         }> = [
             {
-                query: from('public.actor').where({ actor_id: 1 }).many(),
+                query: from('actor').where({ actor_id: 1 }).many(),
```

Comment:
For future PRs: would have been nice if you would have done this change first. Most of the diffs in this MR are just public.x to x.

Long diffs makes it harder to spot bugs.